irc/client.go

```diff
@@ -1310,6 +1310,9 @@ func (client *Client) destroy(session *Session) {
 			client.server.connectionLimiter.RemoveClient(flatip.FromNetIP(ip))
 			source = ip.String()
 		}
+		if !shouldDestroy {
+			client.server.snomasks.Send(sno.LocalDisconnects, fmt.Sprintf(ircfmt.Unescape("Client session disconnected for [a:%s] [h:%s] [ip:%s]"), details.accountName, session.rawHostname, session.IP().String()))
```

Comment:
Sorry, one more thing: don't call `IP().String()` here, use `source`.

Reply:
Apologies for the delay, I had missed this. Iirc the github plugin was fixed and can be resintated. 

In my pre-edited message I had warned I did this asking if it matters.  Does it not entail the same `IP` as `session.IP().String()` is it another instance?. Also in that case what about the `rawHostname` I want the hostname to actually be the hostname if host name look ups are enabled

Reply:
So if you don't mind could you tell me what's the difference. Is it referencing a different session instance? also the `rawHostname` field

Reply:
`rawHostname` is fine.

`source` is equivalent in terms of functionality, it just avoids generating the string twice.

Reply:
(test review comment)